pom.xml

```diff
@@ -70,10 +70,11 @@
 
 		<!-- Documentation code -->
 		<module>commandapi-documentation-code</module>
+        <module>commandapi-documentation-velocity-code</module>
```

Comment:
Not a big deal, but the line here uses spaces while the surrounding lines use tabs. I only noticed because of the change line 76->77, where the only change is going from tabs to spaces.